### ISSUE TITLE
JuliaCon 2021 | Day 3 Single Track | Soumith Chintala Keynote

### ISSUE THREAD

**Issue by KZiemian** [JuliaCommunity/YouTubeVideoTimestamps#125]
_JuliaCon 2021 | Day 3 Single Track | Soumith Chintala Keynote_, [https://www.youtube.com/watch?v=6V6jk_OdH-w](https://www.youtube.com/watch?v=6V6jk_OdH-w)

It is very nontechnical talk, while I prefer technical ones, so making timestamps was especially hard to me. Correct every one of my mistakes that you find and change anything you want.

00:00 Welcome!
01:40 Introduction by the speaker
03:19 What is PyTorch?
04:57 Implementation of PyTorch
05:20 Example: neural network in PyTorch
06:34 General principles of PyTorch
07:40 PyTorch team needs collaborate with others
08:42 PyTorch ecosystem
09:16 State of PyTorch in year 2020
10:00 Community and funding of PyTorch
11:49 PyTorch ecosystem
13:03 The journey of PyTorch
13:45 Where we came from?
15:05 Philosophy/principles of open source project
17:17 User and technology centric projects
21:22 Migrating from Torch to PyTorch
25:32 Interactions with other communities
28:52 Scope & risk
35:17 Measurement/metric
38:43 Scaling
40:45 Vertical and horizontal integration
42:17 Q&A: Do you see TorchScript becoming independent programming language?
44:35 Q&A: Do you imagine PyTorch first class front-end for languages other than Python?

**Comment by logankilpatrick** [comment]
Thank you!
